gcloud/pubsub/topic.py

```diff
@@ -39,9 +42,14 @@ class Topic(object):
     :type connection: :class:gcloud.pubsub.connection.Connection
     :param connection: the connection to use.  If not passed,
                         falls back to the default inferred from the
-                        environment.
+
+    :type add_timestamp_to_messages: boolean
+    :param add_timestamp_to_messages: If true, the topic will add a key,
+                        ``timestamp``, to the attributes of each published
```

Comment:
Did you consider a shorter argument name? (Convention is pointing out something) It's quite long. How about `auto_timestamp`?